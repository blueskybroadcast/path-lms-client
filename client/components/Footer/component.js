import React from 'react';

const Footer = () => (
  <div>
    <div className="footer-spacer" />

    <footer className="account">
      <div className="wrapper pull-left">
        <ul className="navigation">
          <li>
            <a className="events" href="/showroom/events">Events</a>
          </li>
          <li>
            <a className="courses" href="/showroom/courses">Programs</a>
          </li>
          <li>
            <a className="searches" href="/showroom/searches">Search</a>
          </li>
          <li>
            <a className="support" href="/showroom/support">Support</a>
          </li>
          <li>
            <a href="/showroom/sign_up">Sign Up</a>
          </li>
          <li className="footer-login">
            <a href="/showroom/sign_in">Sign In</a>
          </li>
        </ul>
      </div>
      <div className="wrapper pull-right">
        <ul className="navigation font-size-08-em">
          <li>
            <a
              className="custom-privacy-policy-url"
              href="http://www.blueskyelearn.com/privacy/"
              target="_blank"
              rel="noopener noreferrer"
            >
              Privacy Policy
            </a>
            &nbsp;|&nbsp;
            <a
              className="custom-cookies-policy-url"
              href="http://www.blueskyelearn.com/cookies/"
              target="_blank"
              rel="noopener noreferrer"
            >
              Cookies Policy
            </a>
          </li>
        </ul>
      </div>
    </footer>
  </div>
);

export default Footer;
