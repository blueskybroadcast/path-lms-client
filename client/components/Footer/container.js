import { connect } from 'react-redux';

import { currentAccountSlugSelector } from '../../selectors/authSelectors';

import Footer from './component';

const mapStateToProps = state => ({
  slug: currentAccountSlugSelector(state)
});

export default connect(mapStateToProps)(Footer);
