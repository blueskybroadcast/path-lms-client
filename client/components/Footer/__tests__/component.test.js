import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../component';

test('should render Footer correctly', () => {
  const wrapper = shallow(
    <Footer slug="testing" />
  );
  expect(wrapper).toMatchSnapshot();
});
