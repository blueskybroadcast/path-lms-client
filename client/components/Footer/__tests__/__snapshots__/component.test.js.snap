// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Footer correctly 1`] = `
<div>
  <div
    className="footer-spacer"
  />
  <footer
    className="account"
  >
    <div
      className="wrapper pull-left"
    >
      <ul
        className="navigation"
      >
        <li>
          <a
            className="events"
            href="/testing/events"
          >
            Events
          </a>
        </li>
        <li>
          <a
            className="courses"
            href="/testing/courses"
          >
            Programs
          </a>
        </li>
        <li>
          <a
            className="searches"
            href="/testing/searches"
          >
            Search
          </a>
        </li>
        <li>
          <a
            className="support"
            href="/testing/support"
          >
            Support
          </a>
        </li>
        <li>
          <a
            href="/testing/sign_up"
          >
            Sign Up
          </a>
        </li>
        <li
          className="footer-login"
        >
          <a
            href="/testing/sign_in"
          >
            Sign In
          </a>
        </li>
      </ul>
    </div>
    <div
      className="wrapper pull-right"
    >
      <ul
        className="navigation font-size-08-em"
      >
        <li>
          <a
            className="custom-privacy-policy-url"
            href="http://www.blueskyelearn.com/privacy/"
            rel="noopener noreferrer"
            target="_blank"
          >
            Privacy Policy
          </a>
           | 
          <a
            className="custom-cookies-policy-url"
            href="http://www.blueskyelearn.com/cookies/"
            rel="noopener noreferrer"
            target="_blank"
          >
            Cookies Policy
          </a>
        </li>
      </ul>
    </div>
  </footer>
</div>
`;
